src/software/logger/network_logger.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <g3log/g3log.hpp>
+#include <g3log/loglevels.hpp>
+#include <g3log/logmessage.hpp>
+#include <g3log/logworker.hpp>
+
+#include "software/logger/network_sink.h"
+
+/**
+ * This class acts as a Singleton that's responsible for initializing the logger.
+ */
+class NetworkLoggerSingleton
```

Comment:
This currently seems to not be used, I assume the intention is to have command line arg in `full_system` to switch between using this `NetworkLoggerSingleton` and `LoggerSingleton`?

Reply:
It is meant to be used by the Jetson Nano full_system